src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -269,6 +353,80 @@ components:
         - dispatched
         - not-delivered
 
+    SourceType:
+      title: SourceType
+      description: An enumeration of sources
+      type: string
+      enum:
+        - voter-card
+
+    CertificateLanguage:
+      title: CertificateLanguage
+      description: An enumeration of Certificate languages
+      type: string
+      enum:
+        - cy
+        - en
+      default: en
+
+    GenerateTemporaryCertificateRequest:
+      title: GenerateTemporaryCertificateRequest
+      description: Request body to generate a Temporary Certificate
+      type: object
+      properties:
+        gssCode:
+          type: string
+          example: E12345678
+          minLength: 9
+          maxLength: 9
+          description: GSS code of the ERO responsible for the application
+        sourceType:
+          $ref: '#/components/schemas/SourceType'
+        sourceReference:
+          type: string
+          example: 1f0f76a9a66f438b9bb33070
+          description: Reference in the source application that this Temporary Certificate is for.
+        applicationReference:
+          type: string
+          example: V3JSZC4CRH
+          description: The application reference as known by the citizen. Not guaranteed to be unique.
+        firstName:
+          type: string
+          maxLength: 255
+          description: First name of the Elector
+          example: John
+        middleNames:
+          type: string
+          maxLength: 255
+          description: Middle names of the Elector
+          example: Malcolm
+        surname:
+          type: string
+          maxLength: 255
+          description: Surname of the Elector
+          example: Smith
+        certificateLanguage:
+          $ref: '#/components/schemas/CertificateLanguage'
+        photoLocation:
```

Comment:
As there are a few different photos in the system, possible `certificatePhotoLocation` - but probably not necessary given request context. Although the description could possibly be more explicit.

Reply:
Yep, agreed. I'll do that 👍